Mathlib/RingTheory/Etale/Kaehler.lean

```diff
@@ -219,6 +219,7 @@ def tensorCotangent [alg : Algebra P.Ring Q.Ring] (halg : algebraMap P.Ring Q.Ri
         simp
         rfl }
 
+set_option synthInstance.maxHeartbeats 100000 in
```

Comment:
I think this is the only change that gives me pause here.

Reply:
Yes, that's bad. But these files are bad (because of universe issues, for instance), so maybe making these files better would wash this away.